code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -228,9 +228,16 @@ var/list/ai_list = list()
 			var/area/borg_area
 			for(var/mob/living/silicon/robot/R in connected_robots)
 				borg_area = get_area(R)
+				var/robot_status = "Normal"

```

Comment:
While you are at it, this is a good chance to take RR's suggestion of using "Nominal" instead of "Normal" here!